samcli/local/init/templates/cookiecutter-aws-sam-hello-dotnet/{{cookiecutter.project_name}}/src/HelloWorld/aws-lambda-tools-defaults.json

```diff
@@ -8,9 +8,14 @@
   "profile": "default",
   "region": "us-east-2",
   "configuration": "Release",
+  {%- if cookiecutter.runtime == 'dotnetcore2.0' %}
   "framework": "netcoreapp2.0",
   "function-runtime": "dotnetcore2.0",
+  {%- else %}
+  "function-runtime": "dotnetcore2.1",

```

Comment:
If there becomes a dotnetcore2.2, we will default to dotnetcore2.1.

What if we had defaults to set here and then explicitly checked the runtime for each dotnetcore version:

```
if runtime == dotnetcore2.0:
  # stuff for 2.0 version 
elif runtime == dotnetcore2.1:
  # stuff for 2.1 version
else:
  # default

Reply:
As of now I don't know of any VS defaults and purely following the migration guidelines. Now that you mentioned, I think it's best to explicit check runtime and remove `else` altogether until there are known defaults

Reply:
done